--- origsrc/xesam-glib-0.4.0/Makefile.am	2008-07-21 07:19:22.000000000 -0500
+++ src/xesam-glib-0.4.0/Makefile.am	2008-07-29 02:00:26.093750000 -0500
@@ -14,9 +14,7 @@
 pkgconfigdir = $(libdir)/pkgconfig
 pkgconfig_DATA = xesam-glib.pc
 
-xesam_glibdocdir = ${prefix}/doc/xesam-glib
-
-xesam_glibdoc_DATA = \
+noinst_DATA = \
 	README   \
 	COPYING  \
 	AUTHORS  \
--- origsrc/xesam-glib-0.4.0/xesam-glib/Makefile.am	2008-07-21 07:19:23.000000000 -0500
+++ src/xesam-glib-0.4.0/xesam-glib/Makefile.am	2008-07-29 02:00:46.625000000 -0500
@@ -83,7 +83,7 @@
 	xesam-glib-globals.h
 
 libxesam_glib_la_LDFLAGS = \
-	-version-info 0:0:0 -export-symbols-regex '^xesam_g_.*'
+	-version-info 0:0:0 -no-undefined -export-symbols-regex '^xesam_g_.*'
 
 libxesam_glib_la_LIBADD = \
 	$(XESAM_GLIB_LIBS) -lm
